frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -0,0 +1,307 @@
+<template lang='pug'>
+table.table.table-in-card.c-payments(:class='{"c-payments-edit": paymentsType === "edit"}')
+  thead
+    tr
+      th(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(
+          @change='checkAllpayment'
+          type='checkbox'
+        )
+      th {{ titles.one }}
+      th.c-payments-amount {{ titles.two }}
+      th.c-payments-date {{ titles.three }}
+
+  tbody
+    tr(
+      v-for='(payment, index) in payments'
+      :key='index'
+    )
+      td(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(type='checkbox' v-model='payment.checked')
+      td
+        .c-user
+          avatar-user.c-avatar(:username='payment.to' size='xs')
+          strong.c-name {{payment.to}}
+        .c-user-month(:class='index === 0 ? "has-text-1" : "c-status-danger c-status"') {{ moment(payment.date).format('MMMM') }}
+      td.c-payments-amount(v-if='paymentsType !== "edit"')
+        .c-payments-amount-info(v-if='index === 0 && (!needsIncome && paymentsType === "todo")')
+          i18n.c-payments-amount-text.has-text-1(
+            tag='span'
+            :args='{partial_amount: `<strong class="has-text-0">$20</strong>`, partial_total: currency(payment.amount)}'
+          ) {partial_amount} out of {partial_total}
+
+          i18n.c-status.c-status-primary Partial
+          //- i18n.c-status.c-status-primary(
+          //-   tag='span'
+          //-   role='alert'
+          //-   v-if='payment.hash'
+          //-   :class='payment.paymentClass'
+          //- ) {{ payment.paymentStatusText }}
+
+        strong(v-else) {{currency(payment.amount)}}
+        .c-payments-amount-info(
+          v-if='index === 0 && (needsIncome || paymentsType === "received")'
+          :class='"c-status-warning"'
+        )
+          tooltip.c-name-tooltip(
+            v-if=''
```

Comment:
What's going on with this `v-if`?

Reply:
Logic to implement with real data if the payment is marked as not received. At the moment I just want to display it all the time to look at the design implementation

Reply:
NB: all the condition at the moment are temporary

Reply:
Then there needs to be a `// TODO: ` comment explain exactly what needs to go here